tests/cli.rs

```diff
@@ -476,7 +476,10 @@ fn copy_shell_script() -> color_eyre::Result<()> {
     Ok(())
 }
 
-fn copy_shell_script_rexpect(config_file: PathBuf) -> rexpect::errors::Result<()> {
+#[test]
+fn copy_shell_script_rexpect() -> rexpect::errors::Result<()> {
```

Comment:
The copy tests are the one called `copy` and `copy_shell_script` just above this function. Please revert this change as they call out to this function to test the interactive behavior.